src/patternfly/components/Page/page.scss

```diff
@@ -353,7 +353,7 @@
   // Desktop
   // Collapse nav
   &.pf-m-collapsed {
-    @media screen and (min-width: $pf-global--breakpoint--md) {
+    @media screen {
```

Comment:
Can you remove the wrapping `@media` query, and leave the `max-width` and `overflow` selectors?

Reply:
Makes sense.. done :)